src/board.js

```diff
@@ -1,23 +1,33 @@
-const print = console.log;
+function createBoard(size) {
+    if (size <= 0) {
+        throw new Error(`The board size must be positive: got [${size}].`)
+    }
+    const board = new Array(size);
+    board.forEach((cell, index) => {
+        board[index] = new Array(size);
+    });
+    return board;
+}
 
-function drawBoard(){
-    console.log("hi")
+function size(board) {
+    return board.length;
 }
-function size(grid){
-    print(grid.length)
+
+function drawBoard(board) {
+    throw new Error('We haven\'t done this yet.')
 }
-function draw(cell){
-    if(cell === 1){
+
+function drawCell(cell) {
+    if (cell === 1) {
         console.log("*")
-    }
-    else{
+    } else {
         console.log("_")
     }
-
-    // console.log(cell)
 }
-const board = [1,0,1,0]
-console.log(board)
-drawBoard()
-size(board)
-board.forEach(draw)
\ No newline at end of file
+
+module.exports = {
+    createBoard,
+    size,
+    drawBoard,
+    drawCell
+};
```

Comment:
This is a new thing: [exporting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) functions from a module.

You know that functions can _return_ a value. Files can return stuff too: we call a file that _exports_ stuff a _module_. In this case we're exporting 4 functions that other parts of our program can use.